Libraries/rocSPARSE/preconditioner/csritilu0/main.cpp

```diff
@@ -234,7 +234,8 @@ int main()
 
```

Comment:
std::vector\<double\> data(length_data); always initializes all the values to zero..

Reply:
It doesn't matter what `std::vector<double> data(length_data);` is initialized to. Whatever is there is always overwritten by the copy of `d_data` to the host.